airbyte-integrations/connectors/source-db2-strict-encrypt/src/test/java/io/airbyte/integrations/source/db2_strict_encrypt/Db2JdbcSourceAcceptanceTest.java

```diff
@@ -15,6 +15,7 @@
 import io.airbyte.integrations.source.db2.Db2Source;
 import io.airbyte.integrations.source.jdbc.AbstractJdbcSource;
 import io.airbyte.integrations.source.jdbc.test.JdbcSourceAcceptanceTest;
+import io.airbyte.integrations.source.relationaldb.RelationalDbQueryUtils;
```

Comment:
Not super important but using 

```java
import io.airbyte.integrations.source.relationaldb.RelationalDbQueryUtils.enquoteIdentifier;
```

could make the lines a _little_ shorter, this is just following what I'm seeing in some other files like `MssqlSource.java`